sdk/core/abort-controller/package.json

```diff
@@ -31,21 +31,21 @@
     "unit-test:node": "cross-env TS_NODE_FILES=true TS_NODE_COMPILER_OPTIONS=\"{\\\"module\\\": \\\"commonjs\\\"}\" mocha --require ts-node/register --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --full-trace --no-timeouts test/*.spec.ts",
     "unit-test": "npm run unit-test:node && npm run unit-test:browser"
   },
-  "types": "./types/src/aborter.d.ts",
+  "types": "./types/src/abort-controller.d.ts",
```

Comment:
This doesn't match `publicTrimmedFilePath` in `api-extractor.json`, which is why CI is failing. The lint rule should have caught this. Looks like abort-controller was never using its dtsRollup bundle for types, oops.